lib/fault.ml

```diff
@@ -169,3 +169,48 @@ module Make(A:I) =
             | r -> r
         end)
   end
+
+module Handling : sig
+  type t =
+    | Handled
+    | Fatal
+    | Skip
+
+  val default : t
+  val tags : string list
+  val parse : string -> t option
+  val pp : t -> string
+
+  val is_fatal : t -> bool
+  val is_skip : t -> bool
+end
+=
+struct
+  type t =
+    | Handled
+    | Fatal
+    | Skip
+
+  let default = Handled
```

Comment:
It looks that the default is still handled., while, according to our conversation, it is intended to change to fatal. Is this an oversight?

Reply:
If the change of default impacts the regression test suite, we can change reference files. I do not expect much in the outer world to depend on such a cryptic default behaviour.

Looking back, this default behaviour was adopted because this this is the default behaviour for VM faults observed on top of kvm-unit-tests. I guess this is a rather standard hardware behaviour. However I have been surprised in several occasion and, in the context of our experiments, `fatal` looks a better choice.